app.js

```diff
@@ -0,0 +1,69 @@
+import express from "express";
+import "dotenv/config";
+
+import deleteMovie from "./queries/deleteMovie.js";
+import getAllMovies from "./queries/getAllMovies.js";
+import insertMovie from "./queries/insertMovie.js";
+import updateMovie from "./queries/updateMovie.js";
```

Comment:
I prefer to use named exports rather than default exports. It's all convention and preference, but I find it easier to import multiple functions from a directory using named exports than default.

For example, if you create an `index.js` file inside of `queries` that looks like this
```
export * from './deleteMovie`;
export * from './getAllMovies`;
export * from './insertMovie`;
export * from './updateMovie`;
```

You can now import them all at once inside of `app.js`
```
import { deleteMovie, getAllMovies, insertMovie, updateMovie } from "./queries"
```

Reply:
done 👍